frontend/src/components/Scrubber.js

```diff
@@ -1,39 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
-  Grid,
   Typography,
-  TextField,
   Button,
   Container
 } from "@material-ui/core";
 import { withStyles } from '@material-ui/core/styles';
+import axios from "axios";
+import logo from "../assets/default.png";
 
 function Scrubber(props) {
   const { classes } = props;
+  const [ img, setImg ] = useState(logo);
+
+  const onImgUpload = async event => {
+    const dataURL = await toBase64(event.target.files[0]);
+    const b64 = dataURL.split(',')[1];
+    const url = "https://cors-anywhere.herokuapp.com/https://5pawt634b1.execute-api.us-west-2.amazonaws.com/dev/api/v001/scrub";
+    const res = await axios.post(url, {
+      "data": b64,
+      "ext": ".jpg"
+    });
+    setImg(`data:image/jpeg;base64,${res.data.data}`);
+  }
+
   return (
     <Container component="main" maxWidth="sm">
       <div className={classes.container}>
         <Typography variant="h2" style={{marginTop: "15px"}}>Image Scrubber</Typography>
-        <TextField
-          variant="outlined"
-          margin="normal"
-          required
-          fullWidth
-          label="Image URL"
-          autoFocus
-        />
         <Button
           fullWidth
           variant="contained"
+          component="label"
           color="primary"
+          style={{marginTop: "15px", marginBottom: "15px"}}
         >
-          Srub Image
+          Upload File
+          <input
+            type="file"
+            style={{ display: "none" }}
+            onChange={onImgUpload}
+          />
         </Button>
+        <img style={{maxWidth: "500px", maxHeight: "500px"}} src={img} ></img>
       </div>
     </Container>
   );
 }
 
+const toBase64 = file => new Promise((resolve, reject) => {
+  const reader = new FileReader();
+  reader.readAsDataURL(file);
+  reader.onload = () => resolve(reader.result);
```

Comment:
Stylistic decision, but perhaps this could become this 
```suggestion
  reader.onload = () => resolve(reader.result.split(',')[1]);
```
and remove line 17. However, I'm not sure if that's the "*correct*" way to do it. And in reality I'm it's really just a nit pick and it's fine either way. Still think this is really impressive though :)